watchpack/index.d.ts

```diff
@@ -0,0 +1,54 @@
+// Type definitions for watchpack 1.1
+// Project: https://github.com/webpack/watchpack
+// Definitions by: e-cloud <https://github.com/e-cloud>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { EventEmitter } from 'events';
+import Watcher = require('./Watcher');
+
+declare class Watchpack extends EventEmitter {
+    aggregatedChanges: string[];
+    aggregateTimeout: NodeJS.Timer;
```

Comment:
[Everything I see from the docs](https://github.com/webpack/watchpack/search?utf8=%E2%9C%93&q=aggregateTimeout) seem to indicate that this should be a `number`